lib/compress.js

```diff
@@ -77,6 +77,7 @@ function Compressor(options, false_by_default) {
         screw_ie8     : true,
         sequences     : !false_by_default,
         side_effects  : !false_by_default,
+        switches      : !false_by_default,
```

Comment:
I personally prefer the option name `switch`. But there seems to be a tradition of pluralization in option names. :-)